docs/DeveloperManuals/PluginImplementation.md

```diff
@@ -37,7 +37,7 @@ A plugin mainly consists of a collection of subtasks that can be executed by Dev
 4. [PluginApi](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_api.go) lets a plugin exposes some self-defined APIs
 5. [PluginMigration](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_migration.go) is where a plugin manages its database migrations 
 6. [PluginModel](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_model.go) allows other plugins to get the model information of all database tables of the current plugin through the GetTablesInfo() method.If you need to access Domain Layer Models,please visit [DomainLayerSchema](https://devlake.apache.org/docs/DataModels/DevLakeDomainLayerSchema/)
-7. [PluginBlueprint](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_blueprint.go) is the foundation for Blueprint and Plugin to collaborate to generate a reasonable Pipeline Plan based on User Settings, for example, users may declare that he wanted to collect data from a Github Repo, it implies not only the issues/prs, but also the git-meta-data including commits history, branches, tags, etc, in order to do it and do it fast, leveragin [GitExtractor](../Plugins/gitextractor) is the best and recommended way. So, naturally, The Github/Gitlab/BitBucket plugins would likely to produce 2 concurrent `tasks`, one for itself to collect issues/prs, one for `gitextractor`. In `PluginBlueprintV200`, a new returning value `[]Scope` was added to `MakePipelinePlan` for plugin to declare the `data-scopes` it might produce. For more detail, please consult the [source file](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_blueprint.go)
+7. [PluginBlueprint](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_blueprint.go) is the foundation for Blueprint and Plugin to collaborate to generate a reasonable Pipeline Plan based on User Settings, for example, users may declare that he wanted to collect data from a Github Repo, it implies not only the issues/prs, but also the git-meta-data including commits history, branches, tags, etc, in order to do it and do it fast, leveraging [GitExtractor](../Plugins/gitextractor) is the best and recommended way. So, naturally, The Github/Gitlab/BitBucket plugins would likely to produce 2 concurrent `tasks`, one for itself to collect issues/prs, one for `gitextractor` to collect git-meta-data. For more detail, please consult the [source file](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_blueprint.go)
```

Comment:
Text changes:
1. is the foundation for Blueprint and Plugin to collaborate and generate...
2. For example, a user may declare that he or she wants to collect data from a GitHub repo, which implies that not only the issues/PRs, but also the git-meta-data including commits history, branches, tags, and etc. needs to be collected."
3. In order to do it and do it faster
4. For more details, please read the [source file](https://github.com/apache/incubator-devlake/blob/main/plugins/core/plugin_blueprint.go)